src/main/java/uk/gov/hmcts/divorce/caseworker/event/page/AmendCase.java

```diff
@@ -21,7 +21,7 @@ public void addTo(PageBuilder pageBuilder) {
             .done()
             .complex(CaseData::getApplication)
                 .complex(Application::getMarriageDetails)
-                    .optionalWithLabel(MarriageDetails::getDate,
+                    .mandatoryWithLabel(MarriageDetails::getDate,
```

Comment:
this is weird - was ti a defect that it used to be optional?

Reply:
I'm not sure - I checked with Kim, and she confirmed that this field should not be optional, and now we're adding validation to prevent it being empty it won't be.